tests/datavolume_test.go

```diff
@@ -40,7 +40,7 @@ const (
 	shortTimeout        = 30 * time.Second
 )
 
-var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests", func() {
+var _ = FDescribe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests", func() {
```

Comment:
You need some more changes to make this achievable, here's a reference
https://github.com/kubevirt/containerized-data-importer/pull/2384/files#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162